src/motion/basic_target_follower.cpp

```diff
@@ -11,8 +11,7 @@ constexpr float kAlt = 0.01;
 // Forward velocity of the sub
 constexpr float kSpeed = 0.1;
```

Comment:
Read this from launch file. We want to decide the speed of the submarine when we launch the whole code.